db/migrate/20160922090347_remove_field_region_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveFieldRegionFromUsers < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :users, :region, :integer
```

Comment:
Added non-unique index here. 

Even though this pr grows by suggested off-topics, this needs to be part of this pr.

The index was not `[:region, :userid]`, but actually: `[:userid, :region]`. That means dbms was able to use (and in postgresql case it was using) this index when searching for User of given userid.

Thanks @kbrock for pointing this out. I will make sure to check indexes before my next migration.